nostalgia/sources/abnamro.py

```diff
@@ -0,0 +1,40 @@
+import pandas as pd
+from nostalgia.ndf import NDF
+from nostalgia.times import datetime_from_format
+import re
+import just
+import time
+import numpy as np
+from nostalgia.times import datetime_from_timestamp
+
+def convert_date(date):
+    return datetime_from_format(str(date),"%Y%m%d")
+
+def find_date(x):
+    date_regex = ".*(\d{2}[\.-]\d{2}[\.-]\d{2,4}[\/]\d{2}[\.-]\d{2}).*"
+    value = str(x).replace('\\','\\\\')
+    m = re.match(date_regex, value)
+    if m:
+        return datetime_from_format(m.group(1),"%d.%m.%y/%H.%M")
+
+class AbnAmro(NDF):
+    @classmethod
+    def load(cls, files="~/nostalgia_data/input/abnamro/*.xls", nrows=None):
```

Comment:
The glob should be moved outside of the arguments, just as a constant on the line below :)

Reply:
Yep, thanks